### ISSUE TITLE
train.py

### ISSUE THREAD

**Issue by xuxiuzhi2627** [jaeoh2/Road-Lane-Instance-Segmentation-PyTorch#3]
What are the files in the train path directory? Why is the loss value Nan when I train

**Comment by jaeoh2** [comment]
You can give the train path with argurment option "--train-path" like this:
```
$ python3 train.py --train-path /tuSimple/train_set/ --epoch 100 --batch-size 16 --lr 0.0001 --img-size 224 224
```

You could reduced the learning rate and please refer [here](https://discuss.pytorch.org/t/getting-nan-after-first-iteration-with-custom-loss/25929/11)
